sessionctx/variable/session.go

```diff
@@ -584,3 +543,60 @@ type TableDelta struct {
 	Count   int64
 	ColSize map[int64]int64
 }
+
+// Concurrency defines concurrency values.
+type Concurrency struct {
+	// BuildStatsConcurrencyVar is used to control statistics building concurrency.
+	BuildStatsConcurrencyVar int
+
+	// IndexLookupConcurrency is the number of concurrent index lookup worker.
+	IndexLookupConcurrency int
+
+	// IndexLookupJoinConcurrency is the number of concurrent index lookup join inner worker.
+	IndexLookupJoinConcurrency int
+
+	// DistSQLScanConcurrency is the number of concurrent dist SQL scan worker.
+	DistSQLScanConcurrency int
+
+	// HashJoinConcurrency is the number of concurrent hash join outer worker.
+	HashJoinConcurrency int
+
+	// IndexSerialScanConcurrency is the number of concurrent index serial scan worker.
+	IndexSerialScanConcurrency int
+}
+
+// MemQuota defines memory quota values.
+type MemQuota struct {
+	// MemQuotaQuery defines the memory quota for a query.
+	MemQuotaQuery int64
```

Comment:
Can we remove the 'Mem' prefix of the variables?

Reply:
We embed `MemQuota` in `SessionVars`, so we access `MemQuotaXXX` directly from `SessionVars`, remove `MemQuota` prefix make the variable unclear.